.github/NativeBuildReport.kts

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env kscript
+
+@file:MavenRepository("jcenter","https://jcenter.bintray.com/")
+@file:MavenRepository("maven-central","https://repo1.maven.org/maven2/")
+@file:DependsOn("org.kohsuke:github-api:1.101")
+
+
+import org.kohsuke.github.*
+import java.io.File
+import java.io.IOException
+import java.util.concurrent.TimeUnit
+
+val token = args[0]; 
+val status = args[1];
+
+
+// Handle status. Possible values are success, failure, or cancelled.
+val succeed = status == "success";
+if (status == "cancelled") {
+    println("Job status is `cancelled` - exiting")
+    System.exit(0)
+}
+
+val ISSUE_TITLE = "Quickstarts Native Build failed against Quarkus Master"
+val REPO = "quarkusio/quarkus-quickstarts"
+
+val github = GitHubBuilder().withOAuthToken(token).build()
+val repository = github.getRepository("quarkusio/quarkus-quickstarts")
+val issues = repository.getIssues(GHIssueState.OPEN)
```

Comment:
I wonder for the sake of simplicity if we should not have one issue that is closed and reopened when the quickstarts fail. My main (only reason) is that we can enforce that all the relevant persons are subscribed to the issue and they will get notification through GitHub. This will be more visible than a random new issue created.

Alternatively, we have a list of people we notify (I'd like to be a part of).

Reply:
We can do this, and use a single issue. We just need to be sure that no-one will ever create another issue with the same title.

Reply:
I like this idea, since subscriptions would just work as opposed to have the relevant constantly monitor for new issues

Reply:
I can definitely add [CI] and add the link to the action page (see next comment about the impossibility to retrieve the log from inside the workflow)

Reply:
Sounds good to me

Reply:
New title: `[CI] - Quickstarts Native Build failed against Quarkus Master` label `area/infra`.